local-cli/server/runServer.js

```diff
@@ -69,7 +69,6 @@ function runServer(args, config, readyCallback) {
 
       wsProxy = webSocketProxy.attachToServer(serverInstance, '/debugger-proxy');
       ms = messageSocket.attachToServer(serverInstance, '/message');
-      webSocketProxy.attachToServer(serverInstance, '/devtools');
```

Comment:
Not that I'm aware of. It was used at some point in the past because we used to proxy DevTools connection through the packager port. However we [stopped doing that](https://github.com/facebook/react-native/commit/9a93a3cba47722a590a8912a5ace1c479eb4178a) in favor of a hardcoded different port anyway, so the proxy was not used since then. I verified that contacting `/` directly (instead of `/devtools`) works just fine so [that's what I'm doing now](https://github.com/facebook/react-devtools/blob/309c926365483f62ce2420e0baaa7eef26c76da0/packages/react-devtools-core/src/backend.js#L53).